.github/workflows/reviewdog-workflow.yml

```diff
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: reviewdog-workflow
+name: PR Lint & Test Reviewdog
```

Comment:
The Reviewdog workflow is only a `Lint` type of workflow, imho.

Reply:
Good point. Fixed